spec/models/task_spec.rb

```diff
@@ -28,9 +28,10 @@
 
   describe '#file_no_business!' do
```

Comment:
Can we make a change to this spec to verify the `created_by` is stored when this method is called?